blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+
+## Creating AKS LTS Clusters
+
+Creating an AKS cluster with LTS support is straightforward and uses the same familiar tools and processes you already know. Here's a simple example using Azure CLI:
+
+```bash
+# Create an AKS cluster with LTS support
+az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSLTSCluster \
+    --tier LTS \
+    --kubernetes-version 1.29 \
+    --node-count 3
+```
+
+You can also deploy AKS clusters with LTS support using:
+- **Azure Portal:** Select "Long Term Support" option during cluster creation
+- **ARM Templates/Bicep:** Specify the LTS tier in your infrastructure-as-code deployments
+- **Terraform:** Updated AKS provider supports LTS cluster configuration
+
+**Important:** AKS LTS is available as part of the **Premium tier**. For detailed pricing information, please refer to the [AKS Premium tier pricing](https://azure.microsoft.com/pricing/details/kubernetes-service/).
+
+AKS clusters with LTS support maintain full compatibility with the existing AKS ecosystem, including Azure Monitor, Azure Policy, and all AKS-managed add-ons.
+
+**Industry-Leading Ecosystem Support:** Unlike other cloud providers that typically limit LTS guarantees to the base Kubernetes API, AKS LTS provides comprehensive support for popular add-ons and components that enterprises depend on for production workloads. This includes coordinated breaking change management, timely CVE fixes, and compatibility assurance throughout the LTS lifecycle.
+
+## AKS LTS Version Compatibility
+
+**Every currently supported AKS Kubernetes version is now also available for long term support (LTS).** This means you can immediately access long-term support for your existing clusters without requiring cluster upgrades or migrations.
+
+Whether you're running the latest version or an older supported version, you can transition to LTS support coverage today by upgrading your cluster to the Premium tier.
+
+**Immediate Benefits:**
+- **No forced migrations:** Your existing clusters can adopt LTS support in-place
+- **Complete coverage:** Every supported version gets the same comprehensive LTS treatment
+- **Instant value:** Start benefiting from extended support timelines immediately
+
+When planning your LTS adoption, consider:
+- **Current cluster assessment:** Evaluate which of your existing clusters would benefit most from extended support
+- **Workload criticality:** Prioritize mission-critical production workloads for LTS coverage
+- **Compliance requirements:** Align LTS adoption with your organization's governance and compliance needs
+
+For detailed migration guidance and best practices, refer to our [AKS LTS migration documentation](https://aka.ms/aks/lts-migration) on Microsoft Learn.
+
+## Upgrading Between LTS Versions
```

Comment:
upgrading between LTS versions is covered here. would be also good to add coverage for the community -> LTS and LTS -> community. especially on LTS -> community, if I do it let's say at the 2 year mark as a user I have a lot of questions on what happens? would it be a jump? if yes, with K8s upstream typically focusing more on n-1 to n upgrades, what level of support is provided by AKS on this (as a user I understand there's an element of shared responsibility here, but a high level 1-2 lines on scope of support and the state transitions would help)